lsp-test/lsp-test.cabal

```diff
@@ -99,7 +99,7 @@ test-suite func-test
   main-is:             FuncTest.hs
   hs-source-dirs:      func-test
   type:                exitcode-stdio-1.0
-  build-depends:       base <4.16
+  build-depends:       base 
```

Comment:
should not have the package base bounds?

Reply:
It depends on `lsp-test` which has the bounds - adding them here just means extra work IMO.

Reply:
oh I miss the existing bound in the package lib